### PR TITLE
k8s-app-db-binding blog post 

### PR DESCRIPTION
**Pull request by denisok**

### CODE REVIEW

**Review by dbazhenov — APPROVED** [review]

File: content/blog/2023-01-24-k8s-app-db-binding.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+date: "2023-01-24T00:00:00+00:00"
+tags: ["kubernetes", "operators", "databases", "PMM", "DBaaS", "minikube"]
+authors:
+  - denys_kondratenko
+images:
+  - blog/2023/2/petclinic.png
+slug: k8s-app-db-binding
+---
+
+[dbaas-operator](https://github.com/percona/dbaas-operator) is Yeat Another DBaaS Kubernetes Operator (need to suggest yadbko as a name) that tries to simplify and unify Database Cluster deployments by building a higher abstraction layer on top of [Percona Kubernetes Operators](https://www.percona.com/software/percona-kubernetes-operators).
```
Comment (spron-in) [inline content/blog/2023-01-24-k8s-app-db-binding.md:12]: Not "Yeat" - "Yet"

File: content/blog/2023-01-24-k8s-app-db-binding.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+service/spring-petclinic created
+
+$ kubectl get pods
+NAME                                                 READY   STATUS    RESTARTS      AGE
+spring-petclinic-f7f587c5c-rvq2v                     0/1     CrashLoopBackOff   2 (17s ago)   67s
+```
+
+As we didn't create a binding yet, the application can't connect to the database and thus fails.
+
+Let us connect it and verify it is working:
```
Comment (spron-in) [inline content/blog/2023-01-24-k8s-app-db-binding.md:118]: connect?

**Comment by denisok** [comment]
@dbazhenov let ppl review :) before merging :)

Sergey, I will address your comments

**Comment by dbazhenov** [comment]
> @dbazhenov let ppl review :) before merging :)
> 
> Sergey, I will address your comments

It's too complicated, I trust you :)

Install Grammarly :)

**Comment by denisok** [comment]
> It's too complicated, I trust you :)
we have blog post reviews by experts for percona blogs, would be cool to replicate it here as well if possible. So ppl have a chance to say if it is complete garbage or not :)

> Install Grammarly :)
Funny enough, this blog was done in Grammarly first (enterprise), still some issues :)

**Comment by dbazhenov** [comment]
You got it, I'll read it before I publish it. 
You talked me into it, I will read carefully and run it through Grammarly before I publish it. 

But Sergey is a super expert, it's not fair, he will always find something to get at.

**Pull request merged**